src/main/java/org/primeframework/mvc/security/UserLoginSecurityScheme.java

```diff
@@ -30,6 +31,9 @@
  * @author Brian Pontarelli
  */
 public class UserLoginSecurityScheme implements SecurityScheme {
+
+  private static final Set<HTTPMethod> CSRF_METHODS = Set.of(HTTPMethod.POST, HTTPMethod.PUT, HTTPMethod.PATCH, HTTPMethod.DELETE);
```

Comment:
Another option is to allow these to be overriden by the impl if there is any chance a user of this code may not want to apply CSRF to all of these methods. 

In theory this should be ok, since the `UserLoginSecurityContext` is intended to be browser based, but I don't know all of the use cases in use by this scheme.